mne/cov.py

```diff
@@ -1431,11 +1435,12 @@ def _get_whitener_data(info, noise_cov, picks, diag=False, rank=None,
         noise_cov = cp.deepcopy(noise_cov)
         noise_cov['data'] = np.diag(np.diag(noise_cov['data']))
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings']
```

Comment:
I think I may finally understand the source of my confusion.  It seems to me that there are actually 3 sets of values, two of which happen to be the same:

1. cov estimation: <code>dict(mag=1e15, grad=1e13, eeg=1e6)</code>
2. whitening and rank estimation of cov: <code>dict(mag=1e12, grad=1e11, eeg=1e5)</code>
3. rank estimation of signals: <code>dict(mag=1e15, grad=1e13, eeg=1e6)</code>

Correct?  

If yes, then I propose three dicts in DEFAULTS to be explicit: <code>scalings_cov_est</code>, <code>scalings_cov_rank</code>, and <code>scalings_sig_rank</code>. 

OK?